ape_solidity/compiler.py

```diff
@@ -105,15 +97,45 @@ def import_remapping(self) -> Dict[str, str]:
             if len(item_parts) != 2:
                 raise IncorrectMappingFormatError()
 
-            mapped_path = _try_add_packages_path_prefix(Path(item_parts[1]))
+            suffix = Path(item_parts[1])
```

Comment:
the main change here is that we are de-compiling a cached manifest file rather than just copying over downloaded files.